0.3/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN curl -fSL https://github.com/JanusGraph/janusgraph/releases/download/v${JANU
 COPY conf/janusgraph-berkeleyje-lucene-server.properties conf/log4j-server.properties ${JANUS_HOME}/conf/gremlin-server/
 COPY scripts/remote-connect.groovy ${JANUS_HOME}/scripts/
 
-FROM openjdk:8-jdk
+FROM openjdk:8-jdk-buster
```

Comment:
This change which is modifying the base image seems orthogonal to the overall change described in this PR which is adding tags (and is not mentioned in the PR description). Is the current `openjdk:8-jdk` equivalent to `openjdk:8-jdk-buster` and this is just making it explicit? Or is the goal to choose a more specific (and hence more stable) base image before instituting the tagging policy?

Also, if we're re-considering the base images, what about considering:

* using `slim-buster` instead of `buster` which is smaller
* using `jre` instead of `jdk` since it's smaller

Bottom line: would it make sense to make the base image more explicit and/or smaller, and if you want to make that change _before_ the tagging process which codifies stability guarantees, maybe it makes sense to change the base images as a separate change?

Reply:
> this is just making it explicit? 

Yes

My plan is to make it more specific in follow up pull request. I could do it before. 

I have two ideas todo. 
- Start image as JanusGraph user
- Use jre slim image #75

After this pr is merged we can search for a solution to prepare breaking changes.